### PR TITLE
Implement backend topic import

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Apart from the normal feature branches, this very specifically needs OpenSlides/openslides-backend#1697 in order to work properly.

### CODE REVIEW

**Review by rrenkert — CHANGES_REQUESTED** [review]
* In the fullscreen preview dialog the row content does not fit the table header:
Tagesordnung - OpenSlides - rtf staging openslides com

* Closing this dialog clears the embedded table.
* The preview should disappear after successful import (offer a navigation to the agenda?)

**Review by bastianjoel — APPROVED** [review]

**Review by rrenkert — CHANGES_REQUESTED** [review]
* In full screen the columns "title", "text", and "comment" should use kind of dynamic width.
* After performing an initial import a dialog "Stopped waiting for process(es)" comes up. Refreshing the list view shows all imported items.
* After perfoming an import the user can select a new file to import, but no new preview is shown.

**Review by rrenkert — CHANGES_REQUESTED** [review]
* In full screen the columns "title", "text", and "comment" should use kind of dynamic width.
* After performing an initial import a dialog "Stopped waiting for process(es)" comes up. Refreshing the list view shows all imported items.
* After perfoming an import the user can select a new file to import, but no new preview is shown.

**Review by bastianjoel — APPROVED** [review]

**Comment by luisa-beerboom** [comment]
> * In full screen the columns "title", "text", and "comment" should use kind of dynamic width.
> 
>     * After performing an initial import a dialog "Stopped waiting for process(es)" comes up. Refreshing the list view shows all imported items.
> 
>     * After perfoming an import the user can select a new file to import, but no new preview is shown.

I only fixed the first one, as I can't replicate the others. Are you sure that your instance is working correctly @rrenkert ?

**Comment by rrenkert** [comment]
Looks good so far.

Some minor things:
* Importing duplicated topics shows a misleading tooltip "Duplizieren" for the warning icon.
* After a successful import the summary shows
  
Total topics:	 19
Topics with warnings (will be skipped):	 19

better would be

Total topics:	 19
Topics skipped:	 17
Topics created: 2

> I only fixed the first one, as I can't replicate the others. Are you sure that your instance is working correctly @rrenkert ?
I can still replicate the one with the second file upload:
* select a file and perform the import.
* select the same file again -> nothing happens.

**Review by rrenkert — CHANGES_REQUESTED** [review]
please see my last comment.

**Comment by rrenkert** [comment]
Additional info for the second file upload: Happens in Chromium based browsers.

Another point: When selecting a file for upload a scrollbar appears in the upper card.

File: client/src/app/ui/modules/import-list/components/via-backend-import-list/backend-import-list.component.html [hunk 1]
```diff
@@ -137,7 +137,7 @@ <h2>{{ 'Preview' | translate }}</h2>
             </mat-select>
         </mat-form-field>
     </div>
-    <div>
+    <div style="margin-bottom: 5px">
```
Comment (bastianjoel) [inline client/src/app/ui/modules/import-list/components/via-backend-import-list/backend-import-list.component.html:140]: Not entirely sure but afaik our linter should complain about inline style definitions. Use a class instead if possible.

**Review by bastianjoel — CHANGES_REQUESTED** [review]

**Review by rrenkert — APPROVED** [review]
Works as expected now.

**Review by bastianjoel — APPROVED** [review]

**Pull request merged**